docs/CLI.md

```diff
@@ -194,7 +194,7 @@ Prints the test results in JSON. This mode will send all other test output and u
 
 ### `--outputFile=<filename>`
 
-Write test results to a file when the `--json` option is also specified.
+Write test results to a file when the `--json` option is also specified. The returned JSON structure is documented in [testResultsProcessor](Configuration.md#testResultsProcessor-string)
```

Comment:
```suggestion
Write test results to a file when the `--json` option is also specified. The returned JSON structure is documented in [testResultsProcessor](Configuration.md#testResultsProcessor-string).
```

can you make the same change to the others? 🙂

Reply:
The other one, you mean other cli that return the json like the `--json`

Thank you for the helpful guidance.

Reply:
Ohhhh I have seen the change. sorry let me do that.

Reply:
no, my suggestion of adding a full stop (`.`) at the end of the sentence 🙂

Reply:
Doing that right now. Thank you

Reply:
I have added the full stop. Thank you